apps/bitfeed/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "2.1"
+
+services:
+  web:
+    image: bitfeed/client:2.1.2@sha256:a35ccd4d5149ae62b8b9daa7ddaad9b2fe127ddfc0d0bf3af3646dfb9bae1739
+    restart: on-failure
+    stop_grace_period: 1m
+    depends_on:
+      - "api"
+    environment:
+      TARGET: "umbrel"
+      BACKEND_HOST: "$APP_BITFEED_API_IP"
+      BACKEND_PORT: "$APP_BITFEED_API_PORT"
+    ports:
+      - "$APP_BITFEED_PORT:80"
+    networks:
+      default:
+        ipv4_address: $APP_BITFEED_IP
+
+  api:
+    image: bitfeed/server:2.1.2@sha256:2b5097274b2c433e30cb82b7fdda13c0b856527957ddd15545bd498034be8e51
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      PORT: "$APP_BITFEED_API_PORT"
+      BITCOIN_HOST: "$BITCOIN_IP"
+      BITCOIN_ZMQ_RAWTX_PORT: "$BITCOIN_ZMQ_RAWTX_PORT"
+      BITCOIN_ZMQ_RAWBLOCK_PORT: "$BITCOIN_ZMQ_RAWBLOCK_PORT"
+      BITCOIN_RPC_PORT: "$BITCOIN_RPC_PORT"
+      BITCOIN_RPC_USER: "$BITCOIN_RPC_USER"
+      BITCOIN_RPC_PASS: "$BITCOIN_RPC_PASS"
+    ports:
+      - "$APP_BITFEED_API_PORT:$APP_BITFEED_API_PORT"
```

Comment:
Does this port need to be exposed remotely over Tor or to the local network?

If not we can remove this block, it'll still be exposed to the `bitfeed/client` container on the private internal network.

Reply:
Yeah it only needs to be accessible to the web container, I'll remove that block.